brigade/static/scss/style.scss

```diff
@@ -26,3 +26,4 @@
 // Pages
 @import 'pages/home';
 @import 'pages/projects';
+@import 'pages/brigade';
```

Comment:
I wonder if we should call it `brigade-profile` instead of `brigade`, just to make it crystal clear? It's possible we'll also have a `brigade-listing` or `brigade-index` at some point too.

Reply:
Good point, "brigade" is overloaded a million ways. I'll address.

Reply:
I ended up removing this stylesheet since I could just add a list variant for this.